### ISSUE TITLE
[a11y]: toggle component does not appear in high contrast mode 

### ISSUE THREAD

**Issue by jingx-xue** [carbon-design-system/carbon#10458]
### Package

carbon-components-react

### Browser

Chrome

### Operating System

Windows

### Package version

10.43.0

### React version

16.8.6

### Automated testing tool and ruleset

 Windows ease of access

### Description

Without high contrast:

![image](https://user-images.githubusercontent.com/48215968/149830533-f404d5a6-16c7-486d-b404-135d6dda5288.png)

With high contrast:

![image](https://user-images.githubusercontent.com/48215968/149830548-775d90b4-9ae9-4feb-8eea-362882149508.png)

### CodeSandbox example

https://codesandbox.io/s/suspicious-williamson-orf0u

### Steps to reproduce

Enter Windows ease of access high contrast mode
Run accessibility tool

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by joshblack** [comment]
cc @dakahn is this included in some of your HCM fixes or should we add this to a future sprint? 👀

**Comment by tw15egan** [comment]
Closed via https://github.com/carbon-design-system/carbon/pull/12100
